### ISSUE TITLE
Timed out

### ISSUE THREAD

**Issue by salvation-z** [ChaosinaCan/vscode-remote-x11#11]
@ChaosinaCan Hi, i am very glad to use this extension, my local vscode is runing on windows10, while my remote server's system is ubuntu 18.04. When i use this extension, i got this:
```
Setting up display for remote "ssh-remote".
Connecting to SSH xx@xxx.xx.xxx.xx port 22
Error: Timed out while waiting for handshake
```
I have just installed VcXsrv on my computer and double checked the rsa setting and timeout setting(was set to 0). I have on idea how to deal with this problem,  and i need your help.
Before using this extension, I use mobaXterm and it works fine.

If any other information is needed, please notice me~
Thanks a lot

**Comment by ChaosinaCan** [comment]
Could you please provide the output for "Remote X11 (SSH)" as well? I'm not sure what could be causing that.

The timeout setting in my extension is for waiting for the SSH shell to print the value of the `DISPLAY` variable after it's already connected. Hopefully the logs for the SSH extension will provide more info as to why it couldn't connect. If not, there is something I can add to get more verbose logging.

**Comment by salvation-z** [comment]
Thank you for replying~
They provide the same error
```
Connecting to xx@xxx.xx.xxx.xx port 22
Error: Timed out while waiting for handshake
```

**Comment by ChaosinaCan** [comment]
I'll add a setting to print more debug info soon, which should give us more info on why it's timing out.

**Comment by ChaosinaCan** [comment]
I just published version 1.3.2 of the SSH extension, which adds a `remoteX11.SSH.verboseLogging` setting. Please enable that setting, then reload the window and check the output for "Remote X11 (SSH)". It should now print out things such as which authentication algorithms it's trying.

Also, make sure the IP address and port in the "connecting to..." are correct. The automatically determined address and port may be incorrect if you're using things like port forwarding. If they aren't correct, set the `remoteX11.SSH.host` and/or `remoteX11.SSH.port` settings to fix them.

**Comment by salvation-z** [comment]
Thanks again for your help. My IP address and the port are correct. The output of the Remote X11(SSH) is as following:
```
DEBUG: Local ident: 'SSH-2.0-ssh2js0.4.10'
DEBUG: Client: Trying 172.17.224.60 on port 22 ...
```
